### PR TITLE
Insertion sort

### PR DESCRIPTION
**Pull request by lhchandler4**
- [ X] Top-level README “Table of Contents” is updated
 - [ X] Feature tasks for this challenge are completed
 - [ X] Unit tests written and passing
     - [ X] “Happy Path” - Expected outcome
     - [ X] Expected failure
     - [ X] Edge Case (if applicable/obvious)
 - [ X] README for this challenge is complete
     - [ X] Summary, Description, Approach & Efficiency, Solution
     - [ X] Link to code
     - [ X] Picture of whiteboard

### CODE REVIEW

File: challenges/src/main/java/Java/InsertionSort/InsertionSort.java [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+            throw new IllegalArgumentException("Array Is Empty");
+        }
+        for(int i = 1; i < arr.length; i++){
+            int j = i - 1;
+            int tempArr = arr[i];
+
+            while(j >= 0 && tempArr < arr[j]){
+                arr[j + 1] = arr[j];
+                j = j-1;
+
```
Comment (evanslaton) [inline challenges/src/main/java/Java/InsertionSort/InsertionSort.java:16]: empty line!
